packages/connect/src/transactions/types.ts

```diff
@@ -98,7 +102,9 @@ export interface STXTransferBase extends TxBase {
 export interface STXTransferOptions extends STXTransferBase {
   authOrigin?: string;
   userSession?: UserSession;
+  /** DEPRECATED: use `onFinish` */
```

Comment:
Does this need to be `@deprecated` to get the strikethrough? 🤔

Reply:
Yes! I'll update.